homeassistant/components/flume/coordinator.py

```diff
@@ -109,9 +111,13 @@ def _update_lists(self):
                 continue
             device_id = notification["device_id"]
             rule = notification["extra"]["event_rule_name"]
-            active_notifications_by_device.setdefault(device_id, set()).add(rule)
+            time = parse_datetime(notification["created_datetime"])
+            age = utcnow() - (utcnow() - time)
```

Comment:
Please avoid calling `utcnow()` twice.

Reply:
Well that seems like a I wasn't paying attention:  0 - time or -1 * time should suffice.